### PR TITLE
Make consul based dynamic configuration possible

### PR DESCRIPTION
**Pull request by lalyos**

### CODE REVIEW

**Comment by doktoric** [comment]
Missing jira id in commit messages

**Comment by lalyos** [comment]
@keyki @doktoric  please

File: cloud-common/build.gradle [hunk 1]
```diff
@@ -20,4 +20,5 @@ dependencies {
     compile group: 'commons-io', name: 'commons-io', version: '2.4'
     compile group: 'com.google.guava', name: 'guava', version: guavaVersion
     compile group: 'org.freemarker',                name: 'freemarker',                     version: freemarkerVersion
+    runtime('org.springframework.cloud:spring-cloud-starter-consul-config:1.0.0.BUILD-SNAPSHOT')
```
Comment (doktoric) [inline cloud-common/build.gradle]: please use the correct dependency definition format:
runtime group: x, name: y, version: z

**Comment by lalyos** [comment]
@doktoric dependency syntax is fixed

**Pull request merged**